examples/core/transform-feedback/app.js

```diff
@@ -119,7 +119,9 @@ const animationLoop = new AnimationLoop({
 
   onRender({gl, time, renderModel, transformModel}) {
 
-    transformModel.transform();
+    renderModel.vertexArray.unbindBuffers(() => {
```

Comment:
So the burden is on the user to unbind the buffers? Is there a way this can be handled by the Transform class?

Reply:
Should the user directly manipulate `model.vertexArray` at all?

Reply:
I think we should make sure this works in deck.gl then we can brainstorm about the right interface. We could for instance have a `unbind: [models]` parameter to transform that unbinds the vertex arrays of any supplied models.

Or we could just decide to always keep vertex arrays unbound except when we render and effectively return the performance and behavior pre-v6, but that would not be my preference.